qa/tasks/cephfs/test_volumes.py

```diff
@@ -136,6 +136,167 @@ def test_subvolume_create_and_rm(self):
         # verify trash dir is clean
         self._wait_for_trash_empty()
 
+    def test_subvolume_expand(self):
+        # tests if the subvolume can grow
```

Comment:
The style elsewhere in the cephfs tests is to put the test help text / documentation in a string at the start of the method. (I don't know what uses it but I find it helpful.)

So let's change this comment into a string like:

````
"""
That the subvolume can grow in size.
"""
````

And do that for the test of these test cases too.